lib/fileio/commands/baseExportCommand.h

```diff
@@ -0,0 +1,109 @@
+    // and some that are defined by this command itself.
+    // All short forms of the Maya flag names are defined here.
+    // All long forms of flags defined by the command are also here.
+    // All long forms of flags defined by the Arg Tokens are queried
+    // for and set when creating the MSyntax object.
+    // Derived classes can use the short forms of the flags when
+    // calling Maya functions like argData.isFlagSet()
+    //
+    // The list of short forms of flags defined as Arg Tokens:
+    static constexpr auto kDefaultMeshSchemeFlag = "dms";
+    static constexpr auto kExportColorSetsFlag = "cls";
+    static constexpr auto kExportUVsFlag = "uvs";
+    static constexpr auto kEulerFilterFlag = "ef";
+    static constexpr auto kExportVisibilityFlag = "vis";
+    static constexpr auto kExportInstancesFlag = "ein";
+    static constexpr auto kMergeTransformAndShapeFlag = "mt";
+    static constexpr auto kStripNamespacesFlag = "sn";
+    static constexpr auto kExportRefsAsInstFlag = "eri";
+    static constexpr auto kExportDisplayColorFlag = "dsp";
+    static constexpr auto kShadingModeFlag = "shd";
+    static constexpr auto kMaterialsScopeNameFlag = "msn";
+    static constexpr auto kExportMaterialCollectionsFlag = "mcs";
+    static constexpr auto kMaterialCollectionsPathFlag = "mcp";
+    static constexpr auto kExportCollectionBasedBindingsFlag = "cbb";
+    static constexpr auto kNormalizeNurbsFlag = "nnu";
+    static constexpr auto kExportReferenceObjectsFlag = "ero";
+    static constexpr auto kExportSkelsFlag = "skl";
+    static constexpr auto kExportSkinFlag = "skn";
+    static constexpr auto kParentScopeFlag = "psc";
+    static constexpr auto kRenderableOnlyFlag = "ro";
+    static constexpr auto kDefaultCamerasFlag = "dc";
+    static constexpr auto kRenderLayerModeFlag = "rlm";
+    static constexpr auto kKindFlag = "k";
+    static constexpr auto kCompatibilityFlag = "com";
+    static constexpr auto kChaserFlag = "chr";
+    static constexpr auto kChaserArgsFlag = "cha";
+    static constexpr auto kMelPerFrameCallbackFlag = "mfc";
+    static constexpr auto kMelPostCallbackFlag = "mpc";
+    static constexpr auto kPythonPerFrameCallbackFlag = "pfc";
+    static constexpr auto kPythonPostCallbackFlag = "ppc";
+    static constexpr auto kVerboseFlag = "v";
+
+    // Short and Long forms of flags defined by this command itself:
+    static constexpr auto kAppendFlag = "a";
+    static constexpr auto kAppendFlagLong = "append";
+    static constexpr auto kFilterTypesFlag = "ft";
+    static constexpr auto kFilterTypesFlagLong = "filterTypes";
+    static constexpr auto kFileFlag = "f";
+    static constexpr auto kFileFlagLong = "file";
+    static constexpr auto kSelectionFlag = "sl";
+    static constexpr auto kSelectionFlagLong = "selection";
+    static constexpr auto kFrameSampleFlag = "fs";
+    static constexpr auto kFrameSampleFlagLong = "frameSample";
+    static constexpr auto kFrameStrideFlag = "fst";
```

Comment:
Like I mention in my commit message, this short form previously overlapped with the short for of -filterTypes.  Changing this to -fst fixes that.